<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STMessageItemClass</class>
 <widget class="QWidget" name="STMessageItemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STMessageItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(255, 255, 255);}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame#frame {background-color: #ffffff;border: 1px solid #e3e3e3;}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="MaskLabel" name="lblPic">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">border:0px;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblName">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(67, 69, 85);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMessage">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(147, 147, 147);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbAgree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>40</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:pressed {
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:!pressed {
	
	background-color: rgb(109, 206, 160);
}</string>
           </property>
           <property name="text">
            <string>同意</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRefuse">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:pressed {
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:!pressed {
	background-color: rgb(255, 133, 103);
}</string>
           </property>
           <property name="text">
            <string>拒绝</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDelete">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>18</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:pressed {
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:!pressed {
	background-color: rgb(255, 133, 103);
}</string>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaskLabel</class>
   <extends>QLabel</extends>
   <header location="global">masklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/softterminal.qrc"/>
 </resources>
 <connections/>
</ui>
